mypy/sharedparse.py

```diff
@@ -96,3 +98,15 @@ def special_function_elide_names(name: str) -> bool:
 
 def argument_elide_name(name: Union[str, Tuple, None]) -> bool:
     return isinstance(name, str) and name.startswith("__")
+
+
+def _is_overload_decorator(dec):
+    if isinstance(dec, NameExpr) and dec.name in {'overload', 'property', 'abstractproperty'}:
```

Comment:
Can you add a comment explaining why properties are also considered overloads? Is this just because the same parse tree structure is used to support a sequence of `@property`, `@f.getter` and/or `@f.deleter`?